examples/asr/asr_vad/feature_rttm_to_text.py

```diff
@@ -0,0 +1,381 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Given audios or their pre-processed features, as well as their RTTM files,
+peform automatic speech recognition (ASR), and save predicted transcriptions
+in a new manifest.
+
```

Comment:
Are you supporting audio manifests here so that users can make use of the profiling feature? With the file named specifically for feature_rttm, this looks unnecessary. We should have a wrapper script  speech_to_vad_to_text.py that calls both the scripts here and have an option to turn off VAD. For using vAD+ASR looks like we need to run two different scripts now, which is kind of painful.

Reply:
agree. Let's try to get rid of `feature` and merge these to one script.  Ideally we should treat vad as an option in [transcribe_speech.py](https://github.com/NVIDIA/NeMo/blob/main/examples/asr/transcribe_speech.py). But let's discuss offline.

Reply:
The reason I separate two scripts is to allow more flexibility. The first script is for VAD, and users can try different post-processing hyper-parameters with it, without invoking anything from ASR. The second script allows users to try different ASR models with the same VAD output, with the options to use RTTM and/or audio features as input. Also, separating the two scripts allows users to use VAD outputs from non-NeMo models.